build_library/vm_image_util.sh

```diff
@@ -215,11 +215,13 @@ IMG_parallels_DISK_LAYOUT=vm
 IMG_parallels_CONF_FORMAT=pvs
 
 ## ami
-IMG_ami_OEM_PACKAGE=oem-ec2-compat
-IMG_ami_OEM_USE=ec2
 IMG_ami_vmdk_DISK_FORMAT=vmdk_stream
-IMG_ami_vmdk_OEM_PACKAGE=oem-ec2-compat
 IMG_ami_vmdk_OEM_USE=ec2
+IMG_ami_vmdk_OEM_PACKAGE=common-oem-files
+IMG_ami_vmdk_SYSEXT=oem-ec2
+IMG_ami_OEM_USE=ec2
```

Comment:
```suggestion
IMG_ami_OEM_USE=ami
```
We should not change the `ID=` of `oem-release`